docs/packaging/monitoring2.yml.md

```diff
@@ -0,0 +1,139 @@
+
+## "security" fields
+
+| Field    | Type            | Required ? | Description                                                                         |
+| -------- | --------------- | ---------- | ----------------------------------------------------------------------------------- |
+| `cpe`    | list            | Yes        | List of `vendor:product` pairs from a full CPE name. CVEs for a given project may be published under more than one CPE; include more than one if that is likely.                            |
+| `ignore` | list of strings | No         | List of specific CVE identifiers which can be ignored. Todo: do we include reasons? |
+
+### What is a CPE Name?
+
+A [CPE Name](https://en.wikipedia.org/wiki/Common_Platform_Enumeration) is a structured naming scheme for information technology systems, primarily used to search for CVEs. A full CPE Name contains redundant information we can ignore, we are only interested in `vendor` and `product`
+
+For example, `cpe:2.3:a:systemd_project:systemd` is the CPE for the _vendor_ `systemd_project`, and the _product_ `systemd`
+
+### Finding the CPE Name
+
+The easiest way to search for CPE Names is with the following command; replacing `systemd` with your search term:
+
+```bash
+curl -s -X POST https://cpe-guesser.cve-search.org/search -d "{\"query\": [\"systemd\"]}" | jq .
+```
+
+If you have our [helper functions](/docs/packaging/prepare-for-packaging#set-up-monorepo-helper-functions-optional) installed, you can use the following command:
+
+```bash
+cpesearch systemd
+```
+
+
+:::tip
+
+Convert your search term to lower case and try variations on the search term if you get no results. The CPE search is not a "fuzzy" search.
+
+:::
+
+The command returns the following:
+
+```text
+[
+  [
+    49192,
+    "cpe:2.3:a:ubuntu_developers:systemd"
+  ],
+  [
+    116392,
+    "cpe:2.3:a:lennart_poettering:systemd"
+  ],
+  [
+    120506,
+    "cpe:2.3:a:freedesktop:systemd"
+  ],
+  [
+    120627,
+    "cpe:2.3:a:systemd_project:systemd"
+  ]
+]
+```
+
+Ignore the numerical ids, let's walk through the CPEs by vendor:
+
+- `ubuntu_developers` is for `systemd` patched by Ubuntu; we can ignore it
+- `lennart_poettering` is for the main `systemd` developer and is probably a bleeding edge vendor; ignore it
+- `freedesktop` is from freedesktop.org and is a good candidate, so we add it
+- `systemd_project` is a good candidate, so we add it
+
+### No known CPE
+
+It is possible that an established product has not had a security advisory in the past, and therefore **does not have a CPE**; please ask on Matrix if you are unsure. In that case, include an empty `security` and `cpe` field with a comment in the following format:
```

Comment:
- Split this in two paragraphs.
- I think the recommendation of asking in Matrix can be in an admonition.

```suggestion
If an established product hasn't had a security advisory in the past, it might not have a CPE. 

In that case, include an empty `security` and `cpe` field with a comment in the following format:
```